src/views/Main/styles.module.css

```diff
@@ -1,37 +1,41 @@
-@import url("../../styles/base.css");
-@import url("../../styles/queries.css");
-
-.wrapper {
-  overflow-y: scroll;
-  display: flex;
-  margin: 0;
-
-  height: 100vh;
-  -webkit-box-orient: horizontal;
-  -o-box-orient: horizontal;
-
-  flex-direction: column;
-
-  @media (--screen-phone-lg) {
-    flex-direction: row;
-  }
+* {
+  font-family: 'Lato', 'Helvetica Neue', 'Open Sans', sans-serif;
+	font-size: 13px;
```

Comment:
```
* {
  font-family: 'Lato', 'Helvetica Neue', 'Open Sans', sans-serif;
      font-size: 13px;
```

`font-family` is correctly indented. the rest below it are indented way too far in... should be just 2 spaces.